source/extensions/common/wasm/wasm.cc

```diff
@@ -504,6 +504,21 @@ void replaceHeader(Http::HeaderMap* map, absl::string_view key, absl::string_vie
     map->addCopy(lower_key, std::string(value));
 }
 
+const uint8_t* decodeVarint(const uint8_t* start, const uint8_t* end, uint32_t* out) {
+  int ret = 0;
+  while (start < end && *start >= 128) {
+    ret |= (*start & 0x7f);
+    ret >>= 7;
+    start++;
+  }
+  if (start < end) {
+    ret |= (*start & 0x7f);
```

Comment:
add an `ASSERT(*start > 0x7F)` ?

Reply:
IMO not necessary.  Also I don't see a great number of such asserts in the envoy codebase so I don't believe it is the style.  The asserts I see seem to be concentrated in particularly complex code (e.g. the connection manager).

Reply:
Sure. Ignore my comment. Please follow the style.